app/Http/Controllers/DailyMenuController.php

```diff
@@ -3,8 +3,93 @@
 namespace App\Http\Controllers;
 
 use Illuminate\Http\Request;
+use App\DailyMenu;
+use App\Food;
 
 class DailyMenuController extends Controller
 {
-    //
+    /**
+     * The Daily Menu implementation.
+     *
+     * @var DailyMenu
+     */
+    protected $dailyMenus;
+    /**
+     * Create a new controller instance.
+     *
+     * @param DailyMenu $dailyMenus instance of DailyMenu
+     *
+     * @return void
+     */
+    public function __construct(DailyMenu $dailyMenus)
+    {
+        $this->dailyMenus = $dailyMenus;
+    }
+
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        $listDailyMenu = $this->dailyMenus->getAll();
+
+        return view('daily_menus.index', ['listDailyMenu' => $listDailyMenu]);
+    }
+
+    /**
+     * Display the specified resource.
+     *
+     * @param string $date The date to get menu to show
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function show($date)
+    {
+        $menuOnDate = $this->dailyMenus->getMenu($date);
+
+        return view('daily_menus.show', ['menuOnDate' => $menuOnDate, 'date' => $date]);
+    }
+
+    /**
+     * Update item in Menu List on Date
+     *
+     * @param Request $request The request message from Ajax request
+     *
+     * @return Response
+     */
+    public function updateMenuItem(Request $request)
+    {
+        $menuItem = $request->data[0];
```

Comment:
i use send data by Ajax request, so i have validated this date in javascript before it was sent